FAQ.md

```diff
@@ -31,22 +30,34 @@ This is especially problematic for PDB files from X-ray crystallography which ty
 For our purposes here, however, we assume you begin with the coordinates of all atoms present and the full topology of your system.
 
 Given a PDB file of a hypothetical biomolecular system of interest containing a small molecule, there are several routes available to you for treating the small molecule present:
-- Use a cheminformatics toolkit (see above) to infer bond orders
+- Use a cheminformatics toolkit (see below) to infer bond orders
 - Identify your ligand from a database; e.g. if it is in the Protein Data Bank (PDB), it will be present in the [Ligand Expo](http://ligand-expo.rcsb.org) meaning that it has a database entry and code you can use to look up its putative chemical identity
 - Identify your ligand by name or SMILES string (or similar) from the literature or your collaborators
 
+## What about starting from an XYZ file?
+
+XYZ files generally only contain elements and positions, and are therefore similar in content to PDB files. See the above section "What about starting from a PDB file?" for more information.
+
 ## What do you recommend as a starting point?
 
 For application of SMIRNOFF force fields, we recommend that you begin your work with formats which provide the chemical identity of your small molecule (including formal charge and bond order).
 This means we recommend one of the following or equivalent:
-- A `.mol2` file or files for the molecules comprising your system, with correct bond orders and formal charges. (Note: Do NOT generate this from a simulation package or tool which does not have access to bond order information; you may end up with a `.mol2` file, but the bond orders will be incorrect)
+- A `.mol`, `.sdf`, or `.mol2` file or files for the molecules comprising your system, with correct bond orders and formal charges. (Note: Do NOT generate this from a simulation package or tool which does not have access to bond order information; you may end up with a correct-seeming file, but the bond orders will be incorrect)
 - Isomeric SMILES strings for the components of your system
 - InCHI strings for the components of your system
 - Chemical Identity Registry numbers for the components of your system
 - IUPAC names for the components of your system
 
 Essentially, anything which provides the full identity of what you want to simulate (including stereochemistry) should work, though it may require more or less work to get it into an acceptable format.
 
+## I understand the risks and want to perform bond and formal charge inference anyway
+
+If you are unable to provide a molecule in the formats recommended above and want to attempt to infer the bond orders and atomic formal charges, there are tools available elsewhere that can provide guesses for this problem. These tools are not perfect, and the inference problem itself is poorly defined, so you should review each output closely. Some tools we know of include:
```

Comment:
What about stereochemistry? I often include stereochemistry when I describe the difference between representations. But I'm not sure if any of these tools do that?

Reply:
I left it out here for the sake of conciseness, but what I'd say with more words is "in an XYZ file you're totally missing the bond orders and formal charges - they're neither directly readable nor inferrable from the file. But you DO have stereo in there, since the coordinates have all the info needed to unambiguously assign it once the connectivity is perceived". So the user will still probably want to run `OE/RDK.PerceiveStereoFrom3D` to avoid stereo warnings, but that's all separate from the "scary guessing" topic for this point.

Reply:
I think if we're saying "review each output closely", it might be nice to give a brief checklist of things to check - stereochemistry, bond order, partial charges, and whatever I've forgotten (which proves my point :wink:)

Reply:
Good call. I've linked to the "Core concepts" page here, which starts with a discussion of what's in a Molecule